app/src/main/res/values-be/strings.xml

```diff
@@ -20,6 +20,20 @@
     <string name="unpin_conversation">Адмацаваць</string>
     <string name="forward_message">Пераслаць</string>
     <string name="compress_error">Немагчыма сціснуць выяву да выбранага памеру</string>
+    <!-- vCard-->
+    <plurals name="view_contact">
+        <item quantity="one">View contact</item>
+        <item quantity="other">View contacts</item>
+    </plurals>
+    <plurals name="and_other_contacts">
+        <item quantity="one">and %d other</item>
+        <item quantity="other">and %d others</item>
+    </plurals>
+    <string name="contact_info">Contact info</string>
+    <string name="notes">Notes</string>
```

Comment:
there is a Notes string in Simple Commons, will add Anniversary and Birthday from Simple Contacts into Simple Commons too

Reply:
all right, removed "Notes" duplication. Will remove birthday and anniversary too once they are added